run.py

```diff
@@ -16,6 +16,6 @@
         logger.info('* Cloud Foundry port "{}"'.format(port))
         logger.info('* Cloud Foundry protocol "{}"'.format(protocol))
     else:
-        port = os.getenv('FS_DEV_PORT', 5001)
+        port = os.getenv('FS_DEV_PORT', 8080)
```

Comment:
We should probably not change this as 5001 is what most devs will be used to for frontstage at the moment